ansible/provision-pulsar-aws.tf

```diff
@@ -0,0 +1,117 @@
+variable "public_key_path" {
+  description = <<DESCRIPTION
+Path to the SSH public key to be used for authentication.
+Ensure this keypair is added to your local SSH agent so provisioners can
+connect.
+
+Example: ~/.ssh/terraform.pub
+DESCRIPTION
+}
+
+variable "key_name" {
+  description = "Desired name of AWS key pair"
+}
+
+variable "region" {
+    default = "us-west-2"
+}
+
+variable "ami" {
+    default = "ami-9fa343e7" // RHEL-7.4
```

Comment:
Why RHEL-7.4 and not Amazon Linux? Since a pulsar deployment likely needs fast nvram disks, driver support for them comes faster on Amazon Linux. I've had issues before where RHEL doesn't recognize all disks.

Reply:
I'll defer to @merlimat on this

Reply:
@afalko  Main reason for preferring RHEL is that the same Ansible deploy script can be reused without changes in different environments (Google cloud, on-prem, etc..). If we stick with Amazon Linux, we need to have 1 Ansible script for each environment.